Tests

EduardBartolovic opened the pull request:
many tests

RedTo commented:
There are also some files which do not really test anything or a skipped. Maybe it would be better to remove them. Just to be clear...

EduardBartolovic commented:
okay i could do it

EduardBartolovic commented:
so i think i fixed all issues. 

I also included my changes to the privacy policy text in this commit.
I just changed some small details too make suitable to or application.

I am not doing anything today anymore. i have headache. I something is not okay than i will do it tommorrow if you want
